src/cloudflare/internal/images-api.ts

```diff
@@ -303,6 +327,165 @@ function chainStreams<T>(streams: ReadableStream<T>[]): ReadableStream<T> {
   return outputStream;
 }
 
+function concatUint8Arrays(a: Uint8Array, b: Uint8Array): Uint8Array {
+  const result = new Uint8Array(a.length + b.length);
+  result.set(a, 0);
+  result.set(b, a.length);
+  return result;
+}
+
+class Base64Error extends Error {
+  public constructor(cause: unknown) {
+    if (cause instanceof Error) {
```

Comment:
instanceof Error checks are incomplete:

```suggestion
    if (Error.isError(cause)) {
```

Reply:
Error.isError doesn't seem to be in the types for some reason, but made this change anyway